docs/guides/python/llama-rag.mdx

```diff
@@ -0,0 +1,309 @@
+---
+description: 'Making LLMs smarter with Dynamic Knowledge Access using Retrieval Augmented Generation'
```

Comment:
Given the knowledge provided in the guide is static, using "Dynamic" might be seen as misleading?